### PR TITLE
workload: initial pass at tracking entities in workload::Space

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -4544,8 +4544,6 @@ void Application::init() {
     _physicsEngine->init();
 
     EntityTreePointer tree = getEntities()->getTree();
-    connect(tree.get(), &EntityTree::deletingEntity, this, &Application::deletingEntity, Qt::QueuedConnection);
-    connect(tree.get(), &EntityTree::addingEntity, this, &Application::addingEntity, Qt::QueuedConnection);
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: These were old stubbery in Appliation.cpp but I decided the EntityTreeRenderer should handle this stuff.

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -4879,14 +4877,16 @@ void Application::setKeyboardFocusEntity(const EntityItemID& entityItemID) {
...
                 }
                 _lastAcceptedKeyPress = usecTimestampNow();
 
-                setKeyboardFocusHighlight(entity->getWorldPosition(), entity->getWorldOrientation(),
-                    entity->getScaledDimensions() * FOCUS_HIGHLIGHT_EXPANSION_FACTOR);
+                auto entity = getEntities()->getEntity(entityId);
+                if (entity) {
+                    setKeyboardFocusHighlight(entity->getWorldPosition(), entity->getWorldOrientation(),
+                        entity->getScaledDimensions() * FOCUS_HIGHLIGHT_EXPANSION_FACTOR);
+                }
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: This fixes a theoretical crash bug that is unrelated to the work at hand.  Maybe we should port to a PR against master.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7339/app_12448_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7340/app_12448_PR.apk

**Comment by samcake** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7342/app_12448_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7345/app_12448_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7345/HighFidelity-Beta-PR12448-9246f6c24a3f279f7bf4399075eb3964788d57af.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7345/HighFidelity-Beta-PR12448-9246f6c24a3f279f7bf4399075eb3964788d57af.exe
